LESSON-02/README.md

```diff
@@ -0,0 +1,189 @@
+# Introduction
+
+In this Lesson we explain the meaning of operators in programming, types
+and usages.
+
+# Lesson Index
+
+* 2 Operators
+* 2.1 Arithmetic Operators
+* 2.2 Comparison operators.
+* 2.3 Other operators.
+* 2.4 Order in operation.
+* 2.5 Testing operators.
+* 2.6 Exercises.
+* 2.7 Thoughts of an operator.
+
+# 2 Operators
+ 
+ Basically are used to perform operations on variables and values, sound as an easy thing
+ but it is a powerful tool inside programming. Most of the time, the needs for transforming date
+ are fulfilled by the combination of actions and operators in the code.
+
+## 2.1 Arithmetic operators.
+  In most programming languages arithmetic operators are used with numeric values to perform common arithmetical operations, such as addition, subtraction, multiplication etc.
+
+  Addition is represented by + symbol
+  
+    <?php 
+    $x = 10;  
+    $y = 6;
+    echo $x + $y;
+    ?>
+  
+  Will show 16 on the screen
+  	
+  Subtraction is represented by - symbol
+  
+    <?php 
+    $x = 10;  
+    $y = 6;
+    echo $x - $y;
+    ?>
+  
+  Will show 4 on the screen
+  
+  Multiplication is represented by - symbol
+    
+    <?php 
+    $x = 10;  
+    $y = 4;
+    echo $x * $y;
+    ?>
+    
+  Will show 40 on the screen	
+  	
+  Division is represented by / symbol
+     
+    <?php 
+    $x = 10;  
+    $y = 2;
+    echo $x / $y;
+    ?>
+     
+  Will show 5 on the screen 	
+ 
+  Modulus is represented by % symbol
+       
+     <?php 
+     $x = 10;  
+     $y = 2;
+     echo $x / $y;
+     ?>
+       
+  Will show 0 on the screen	
+  		
+## 2.2 Comparison operators.
+
+Comparison operators are used to compare two values
+
+  "Equal" operator is represented by ==  it will return true if operands contain the same value
+    
+    <?php
+    $x = 100;  
+    $y = "100";
+    
+    var_dump($x == $y); 
+    ?>
+  	
+  will echo true
+  
+  "Identical" operator is represented by === will return true if $x is equal to $y, and they are of the same type (This is only an operator available in php)
+
+    <?php
+    $x = 100;  
+    $y = "100";
+    
+    var_dump($x == $y); 
+    ?>
+ 
+  will echo false
+
+  "Not equal" operator is represented by !=  or also <>  Returns true if $x is not equal to $y			
+  
+      <?php
+      $x = 100;  
+      $y = 99;
+      
+      var_dump($x != $y); 
+      ?>
+  
+  will echo false
+  
```

Comment:
Sorry... But should be true, no? 100 is not equal 99...

Reply:
Yes you are right!